wallet/src/Pos/Wallet/Web/Methods/Logic.hs

```diff
@@ -140,61 +149,56 @@ getAccountsIncludeUnready
     -> Bool -> Maybe (CId Wal) -> m [CAccount]
 getAccountsIncludeUnready ws mps includeUnready mCAddr = do
     whenJust mCAddr $ \cAddr ->
-      void $ maybeThrow (noWallet cAddr) $
+      void $ maybeThrow (noSuchWallet cAddr) $
         getWalletMetaIncludeUnready ws includeUnready cAddr
     let accIds = maybe (getAccountIds ws) (getWalletAccountIds ws) mCAddr
     let groupedAccIds = fmap reverse $ HM.fromListWith mappend $
                         accIds <&> \acc -> (aiWId acc, [acc])
     concatForM (HM.toList groupedAccIds) $ \(wid, walAccIds) -> do
-      accMod <- txMempoolToModifier ws mps . eskToWalletDecrCredentials =<< findKey wid
+      accMod <- txMempoolToModifier ws mps . keyToWalletDecrCredentials =<< findKey wid
       mapM (getAccountMod ws accMod) walAccIds
   where
-    noWallet cAddr = RequestError $
-        -- TODO No WALLET with id ...
-        -- dunno whether I can fix and not break compatible w/ daedalus
-        sformat ("No account with id "%build%" found") cAddr
+    noSuchWallet cAddr = NoSuchWalletError $ sformat build cAddr
```

Comment:
Oh, I see where the error was changed :) GitHub hid the diff since it was "too large"